src/patternfly/components/Menu/menu.scss

```diff
@@ -121,10 +124,16 @@
   // Drilled in
   --pf-c-menu--m-drilled-in--c-menu__list-item--m-current-path--c-menu--ZIndex: var(--pf-global--ZIndex--2xl);
 
+  // Footer
+  --pf-c-menu__footer--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-menu__footer--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__footer--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-menu__footer--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__footer--BoxShadow: var(--pf-global--BoxShadow--sm-top);
+  --pf-c-menu__footer--MarginBottom: calc(var(--pf-global--spacer--sm) * -1);
```

Comment:
Looks like this is causing a scrollbar to appear

![Screen Shot 2021-04-14 at 9 26 25 AM](https://user-images.githubusercontent.com/5385435/114717867-7d71ba00-9d03-11eb-8fc0-244024991047.png)

Don't think you need it if you're removing padding from `pf-c-menu`

Reply:
@mattnolting updated!